support/ci/specs.sh

```diff
@@ -75,8 +75,8 @@ find /hab/pkgs/hab_travis/hab/ -type f -name hab -exec cp {} ${TEST_BIN_DIR} \;
 find /hab/pkgs/hab_travis/hab-sup/ -type f -name hab-sup -exec cp {} ${TEST_BIN_DIR} \;
 
 
-# TODO
-# https://docs.travis-ci.com/user/pull-requests
+TODO
```

Comment:
I think that was stubbed out to run different test suites against things that landed in master or were run by sentinels vs against PRs, but not sure.